### DISCUSSION TITLE
How to display the beacon while the tooltip is displayed.

### DISCUSSION THREAD

**Discussion by swathi352001** [gilbarbara/react-joyride#996] [Q&A] [answered]
How to display the beacon while the tooltip is displayed. Usually when i hover over the beacon it disappears and the tootlip is opened.Is there any way to display beacon with the tooltip.

**Comment by gilbarbara** [comment, marked as answer]
Hey @swathi352001 

There isn't. The beacon is a visual indication that there is content to be seen; once you click it, the content appears.

**Comment by swathi-eb** [comment]
But I wan't to display both hover and floater.

**Reply by gilbarbara** [reply]
This isn't possible with this library

**Reply by swathi-eb** [reply]
I was able to do that with css but when i hover over the beacon it moves to the top for 2 to 3 seconds. 

my css code
    visibility: visible !important;
    top: 0px !important;
    left: 0px !important;

**Reply by gilbarbara** [reply]
The beacon is unmounted when the tooltip is mounted, and vice-versa.

**Reply by swathi-eb** [reply]
got it could you please let me know how the translate3D is calculated.

**Reply by gilbarbara** [reply]
The react-floater@0.7.9 package handles the positioning using popper.js@1.16.0
